.gitignore

```diff
@@ -148,4 +148,8 @@ cython_debug/
 
 
 # labs34_ds scratch files.
-test_notebook.ipynb
\ No newline at end of file
+test_notebook.ipynb
+
+# saved bert model
+app/saved_model/
```

Comment:
Good idea - we should also double-check with the team that it's ok to add the model to the .gitignore.

Reply:
I think its something that's we actually need to do. Every time someone downloads the model, I believe it uses our GitHub data so the best workaround is to add it gitignore and if they need the model, they can add it locally. I still need to look into the issue a little more to be 100% sure